src/ingest_validation_tests/segmentation_mask_imagesize_validation.py

```diff
@@ -50,29 +56,20 @@ def collect_errors(self, **kwargs) -> List[Optional[str]]:
                 if not data_path.is_absolute():
                     data_path = Path(self.paths[0]).parent / data_path
 
-                for glob_expr in [
-                    "**/*.ome.tif",
-                    "**/*.ome.tiff",
-                    "**/*.OME.TIFF",
-                    "**/*.OME.TIF",
-                ]:
+                for glob_expr in self.files_to_find:
                     for file in data_path.glob(glob_expr):
                         filenames_to_test.append(file)
-                assert len(filenames_to_test) != 1, "Too many or too few files Mask"
 
-                for glob_expr in [
-                    "**/*.ome.tif",
-                    "**/*.ome.tiff",
-                    "**/*.OME.TIFF",
-                    "**/*.OME.TIF",
-                ]:
                     for file in Path(
                         GetParentData(
                             row["parent_dataset_id"], self.globus_token, self.app_context
                         ).get_path()
                     ).glob(glob_expr):
                         parent_filenames_to_test.append(file)
+
+                assert len(filenames_to_test) != 1, "Too many or too few files Mask"
```

Comment:
Reading this again, I'm actually not sure how many files we are expecting--more than 1? I think I misread this as `== 1` before, which I thought made sense.

Would it be reasonable instead to do something like `assert len(filenames_to_test) == len(parent_filenames_to_test), "Mismatched number of files in dataset and parent_dataset directories."` ?

Reply:
You are right, I thought I had it "==", the idea is that a segmask can only happen if the parent dataset is one image, more than 1 image is not allowed. Updated...

Reply:
That's what I read originally so we're both losing it apparently haha. Okay good now I think!